Limelight/ViewControllers/MainFrameViewController.m

```diff
@@ -1046,12 +1045,20 @@ - (void) enableNavigation {
     self.navigationController.navigationBar.topItem.rightBarButtonItem.enabled = YES;
 }
 
+#if TARGET_OS_TV
+- (BOOL)canBecomeFocused {
+    return YES;
```

Comment:
Just curious, what does this do?

Reply:
tvOS has a "focus engine" that's used to move from item to item using the remote's trackpad or the left analog/d-pad of a connected controller.  Returning `YES` here sets a flag that the given element can be be focused by the focus engine.

A couple of links:
https://developer.apple.com/documentation/uikit/uiview/1622584-canbecomefocused?language=objc
https://developer.apple.com/library/archive/documentation/General/Conceptual/AppleTV_PG/WorkingwiththeAppleTVRemote.html#//apple_ref/doc/uid/TP40015241-CH5-SW5

It's worth noting that I found I didn't *need* to override this function for it to work correctly, but per the API, it should still be necessary.